packages/components/src/components/popover/popover.jsx

```diff
@@ -7,16 +7,38 @@ import Icon                            from '../icon';
 class Popover extends React.PureComponent {
     constructor (props) {
         super(props);
-        this.state = { is_open: false };
+        this.state = {
+            is_open       : false,
+            is_bubble_open: false,
+        };
     }
 
     componentDidMount () {
         this.setState({ is_open: this.props.has_error });
     }
 
-    toggleOpen = () => this.setState({ is_open: Boolean(this.props.message) });
+    toggleOpen = () => {
+        this.setState({ is_open: Boolean(this.props.message) });
+    }
+
+    toggleClose = () => {
+        if (this.props.is_bubble_hover_enabled) {
+            setTimeout(() => {
+                // add delay to check if mouse is hovered on popover bubble
```

Comment:
Can we extract it to a named function? e.g. `openIfHovered` or something?

Reply:
@reza-fs you mean rename the `toggleClose` to `openIfHovered` ?